authn-authz/pinniped-login.md

```diff
@@ -0,0 +1,37 @@
+# Login using Pinniped
+
+As a prerequisite the administrator has provided users access to resources via rolebindings. These can be done with the `tanzu rbac` plugin. See [Bind a user or group to a default role](binding.md).
+
+To login to your cluster using Pinniped follow these steps:
+
+1. Generate and distribute kubeconfig to users
+1. Login with provided kubeconfig
+
+## Generate and distribute kubeconfig to users
+
+As an administrator you need to generate the kubeconfig using the following command:
+
+```
+pinniped get kubeconfig --oidc-skip-listen --oidc-skip-browser --kubeconfig-context <your-kubeconfig-context>  > /tmp/concierge-kubeconfig
```

Comment:
Do we need the --oidc-skip-listen --oidc-skip-browser flags?

Reply:
we dont need it. removing..